frontend/src/main.js

```diff
@@ -1,11 +1,12 @@
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 import 'bootstrap/dist/css/bootstrap.css';
-import FlagIcon from 'vue-flag-icon';
```

Comment:
Is there's some internal logic of changes in that file? (as it looks rather as import sorting and some formating)

Reply:
Should I revert that?

Reply:
No, I'm just curious why such changes.

Reply:
I was changing there but I’ve found a simpler solution ;)